frontend/app/routes/_gcweb-app.personal-information.home-address.suggested.tsx

```diff
@@ -0,0 +1,154 @@
+import type { ReactNode } from 'react';
+
+import { type ActionFunctionArgs, type LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { Form, Link, useLoaderData } from '@remix-run/react';
+
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+
+import { Address } from '~/components/address';
+import { InputRadios } from '~/components/input-radios';
+import { getAddressService } from '~/services/address-service.server';
+import { sessionService } from '~/services/session-service.server';
+import { userService } from '~/services/user-service.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('personal-information');
+
+export const handle = {
+  breadcrumbs: [
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.home', to: '/' },
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.personal-information', to: '/personal-information' },
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.suggested-address' },
+  ],
+  i18nNamespaces,
+  pageIdentifier: 'CDCP-0008',
+  pageTitleI18nKey: 'personal-information:address.suggested.page-title',
+};
+
+/**
+ *
+ * loader, runs before rendering html code
+ *
+ */
+export async function loader({ request }: LoaderFunctionArgs) {
+  const userId = await userService.getUserId();
+  const userInfo = await userService.getUserInfo(userId);
+  const homeAddressInfo = await getAddressService().getAddressInfo(userId, userInfo?.homeAddress ?? '');
+  /**
+   * TODO
+   * CHANGE THE SOURCE OF THE SUGGESTED ADDRESS TO WHAT WS ADDRESS SERVICE IS RETURNING INSTEAD OF MAILING ADDRESS
+   */
```

Comment:
It might be worth it to also mention that the "Address entered" isn't the home address on file, which is what is currently being done. 

The "Address entered" should be the address entered on the previous page/form.

Reply:
Also I'll add a nitpick...

``` typescript
/**
 *
 */
```

Is a specific type of comment that has a specific use.. it's [JSDoc](https://jsdoc.app/) and it doesn't make sense to use it in this context. Instead, you can use:

``` typescript
//
//
//
```

or

``` typescript
/*
 *
 */
```

I wouldn't reject a PR over this, but I thought you should know for future PRs.

Reply:
comments have been changed